README.md

```diff
@@ -42,12 +42,12 @@ class UserRegisteredNotification extends Notification
                     ->line('bar');
     }
     
-    public function onSending($channel): void
+    public function onSending(string $channel, $response): void
```

Comment:
According to Laravel's NotificationSent event, `$response` must be nullable here.

Reply:
The $response variable is set to nullable.